indra/util/aws.py

```diff
@@ -99,6 +99,10 @@ def tag_instance(instance_id, **tags):
         logger.info('Adding project tags "%s" to instance %s'
                     % (filtered_tags, instance_id))
         instance.create_tags(Tags=tag_list)
+        vols = instance.volumes.all()
+        for page in vols.pages():
```

Comment:
Can't this be simplified to `for vol in vols` and omit the explicit page traversal? Or even combine it with the previous line, `for vol in instance.volumes.all():`

Reply:
Possibly. This is just how I got it to work in ipython. `vols` didn't seem to be iterable, although I suppose I didn't try just iterating over it. But the only thing that returned an apparent iterable was `vols.pages()`. It seemed obtuse, but it worked. :man_shrugging: